x-pack/plugins/infra/server/routes/inventory_metadata/lib/get_cloud_metadata.ts

```diff
@@ -36,7 +37,18 @@ export const getCloudMetadata = async (
     body: {
       query: {
         bool: {
-          must: [{ match: { 'event.module': model.requiredModule } }],
+          must: [
+            {
+              range: {
+                [sourceConfiguration.fields.timestamp]: {
+                  gte: currentTime - 86400000, // 24 hours ago
+                  lte: currentTime,
+                  format: 'epoch_millis',
+                },
+              },
+            },
+            { match: { 'event.module': model.requiredModule } },
+          ],
```

Comment:
Will this query change make the experience different for some cases now?

Reply:
No, it just limits the range of data to look back in... Previously it look at all the data and stopped when ES found the first document, now this will just look at the last day's worth of data.

Reply:
In the inventory view, what is it looking for in that last day's worth of data? I don't remember exactly how these queries get stitched together so I don't know how this query differs from the one where we look back 5 or 20 intervals.

Reply:
Looks like this is only needed for AWS datasets. I've added another optimization to skip this query if the required module doesn't match `aws`.